config/locales/client.en.yml

```diff
@@ -714,6 +718,7 @@ en:
       user_added_to_group: "%{actor} added you to the group: %{title}"
       user_mentioned: "%{actor} mentioned you in: %{title}"
       user_reminded: "%{actor} is reminding you to vote in: %{title}"
+      comment_announced: "%{actor} mentioned you in: %{title}"
```

Comment:
I wonder if this should be more specific. What do you think about having two messages:
"%{actor} notified everyone in the group" and 
"%{actor} notified everyone in the thread"